usr/include/pkcs11types.h

```diff
@@ -563,8 +565,8 @@ typedef CK_ULONG CK_ATTRIBUTE_TYPE;
 #define CKA_EC_PARAMS          0x00000180
 #define CKA_EC_POINT           0x00000181
 /* The following are new for v2.11 */
-#define CKA_SECONDARY_AUTH     0x00000200
-#define CKA_AUTH_PIN_FLAGS     0x00000201
+#define CKA_SECONDARY_AUTH     0x00000200 /* Deprecated since 2.11 */
```

Comment:
I also found the note that these attributes are now deprecated. But I didn't find any place saying they are deprecated since 2.11. So they were introduced in 2.11 and then immediately deprecated after 2.11? Interesting.

Reply:
In v2.20 header file it states:

```
/* CKA_SECONDARY_AUTH, CKA_AUTH_PIN_FLAGS,
 * are new for v2.10. Deprecated in v2.11 and onwards. */
#define CKA_SECONDARY_AUTH     0x00000200
#define CKA_AUTH_PIN_FLAGS     0x00000201
```